src/styles/_react-switch-button.scss

```diff
@@ -0,0 +1,229 @@
+/**
+ * React Switch Button control style.
```

Comment:
Not a big fan of copying styles to override as often we only need to change a few classes. But if it works 👍